actors/common/src/main/java/org/sunbird/learner/actors/NotesManagementActor.java

```diff
@@ -154,14 +160,13 @@ private void updateNote(Request actorMessage) {
         sender().tell(exception, self());
         return;
       }
-      Map<String, Object> req = (Map<String, Object>) actorMessage.getRequest().get(JsonKey.NOTE);
+      Map<String, Object> req = actorMessage.getRequest();
       req.remove(JsonKey.USER_ID);
       req.remove(JsonKey.COURSE_ID);
       req.remove(JsonKey.CONTENT_ID);
       req.remove(JsonKey.IS_DELETED);
       req.put(JsonKey.ID, noteId);
-      String updatedBy = (String) actorMessage.getRequest().get(JsonKey.REQUESTED_BY);
-      req.put(JsonKey.UPDATED_BY, updatedBy);
+      req.put(JsonKey.UPDATED_BY, userId);
       req.put(JsonKey.UPDATED_DATE, ProjectUtil.getFormattedDate());
 
       Response result =
```

Comment:
Since the map contains context. It may cause error when updating record. This may need to be removed from map.